packages/@react-spectrum/link/test/Link.test.js

```diff
@@ -27,6 +27,7 @@ describe('Link', function () {
   let onPressSpy = jest.fn();
 
   afterEach(() => {
+    cleanup();
```

Comment:
why was this added? it should be enforced now

Reply:
I believe as of now that every call to render appends to document.body, which means when we call `let link = getByRole('link');` we're actually selecting 2 links from different tests. That ends up throwing a warning, and could potential result in false positives in tests.

Reply:
where are we rendering two links?
if you're referring to it.each, i believe every iteration of each is considered it's own test and should get cleanup automatically applied

Reply:
I'm very confused now. I can't reproduce this anymore. I was originally seeing a warning when calling `getByRole`. When debugging, I could see the 2 links, the one from the 2nd-last test, and from the last test. That's what led me down calling `cleanup()` in `afterEach()`. According to the docs it should definitely be calling cleanup automatically in the afterEach block, so I'll remove this addition.